### PR TITLE
Update tooltip.js

### PR DESCRIPTION
**Pull request by Ashishyadav13**
Qlik Cloud: Network charts showing tags when howering

## Description
The hovering is used to show the dimension values. This functionality is not working fine in the Cloud.

[QB-11333](https://jira.qlikdev.com/browse/QB-11333) 

Attached image for Reference.

Screenshot 2022-07-25 at 1 42 29 PM

### CODE REVIEW

File: src/tooltip.js [hunk 1]
```diff
@@ -24,5 +24,5 @@ export function createTooltipHTML({ name, groupNumber, nodeMeasure }) {
   tooltip.appendChild(groupNumberEntry);
   tooltip.appendChild(nodeMeasureEntry);
 
-  return tooltip.innerHTML;
```
Comment (LiKang6688) [inline src/tooltip.js:27]: is any unit test cover it?

Comment (Ashishyadav13) [reply src/tooltip.js:27]: No. for this repo there is no test cases previously written.

Comment (Ashishyadav13) [reply src/tooltip.js:27]: No. for this repo there is no test cases previously written.

Comment (LiKang6688) [reply src/tooltip.js:27]: I suggest that unit tests should be added as we don't know the reason it behaves differently and @Caele also recommends doing that as https://github.com/qlik-oss/sn-network-chart/blob/master/src/sn-paint.js#L56

Comment (Ashishyadav13) [reply src/tooltip.js:27]: There is no test cases are written particular for this “sn-network-chart” repo, so can you please suggest me how and where to write test cases.

Comment (Ashishyadav13) [reply src/tooltip.js:27]: I have added test cases.

Comment (LiKang6688) [reply src/tooltip.js:27]: Nice! Great work

**Review by LiKang6688 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -29,12 +30,16 @@
     "@nebula.js/cli-sense": "1.7.0",
     "@nebula.js/cli-serve": "1.7.0",
     "@nebula.js/stardust": "1.7.0",
+    "@testing-library/jest-dom": "^5.16.4",
     "babel-eslint": "10.1.0",
-    "scriptappy-from-jsdoc": "^0.7.0",
+    "babel-jest": "^28.1.3",
```
Comment (LiKang6688) [inline package.json:35]: Please use the static version, '28.1.3'. and Others as well.

Comment (Ashishyadav13) [reply package.json:35]: ok thank you I will change

File: package.json [hunk 3]
```diff
@@ -29,12 +30,16 @@
     "@nebula.js/cli-sense": "1.7.0",
     "@nebula.js/cli-serve": "1.7.0",
     "@nebula.js/stardust": "1.7.0",
+    "@testing-library/jest-dom": "^5.16.4",
```
Comment (LiKang6688) [inline package.json]: this one as well.

File: package.json [hunk 4]
```diff
@@ -29,12 +30,16 @@
...
-    "scriptappy-from-jsdoc": "^0.7.0",
+    "babel-jest": "28.1.3",
     "eslint": "7.32.0",
+    "jest": "28.1.3",
+    "jest-environment-jsdom": "28.1.3",
+    "scriptappy-from-jsdoc": "0.7.0",
     "shx": "^0.3.3",
-    "vis-network": "9.1.0",
-    "vis-data": "^7.0.0"
+    "vis-data": "^7.0.0",
```
Comment (LiKang6688) [inline package.json:41]: this one as well.

File: package.json [hunk 5]
```diff
@@ -29,12 +30,16 @@
...
     "@nebula.js/stardust": "1.7.0",
+    "@testing-library/jest-dom": "5.16.4",
     "babel-eslint": "10.1.0",
-    "scriptappy-from-jsdoc": "^0.7.0",
+    "babel-jest": "28.1.3",
     "eslint": "7.32.0",
+    "jest": "28.1.3",
+    "jest-environment-jsdom": "28.1.3",
+    "scriptappy-from-jsdoc": "0.7.0",
     "shx": "^0.3.3",
```
Comment (LiKang6688) [inline package.json:40]: this one as well.

File: package.json (hunk 3 above)
Comment (Ashishyadav13) [reply package.json]: changed

File: package.json (hunk 4 above)
Comment (Ashishyadav13) [reply package.json]: changed

File: package.json (hunk 5 above)
Comment (Ashishyadav13) [reply package.json]: changed

File: yarn.lock [hunk 6]
```diff
@@ -1550,13 +2215,47 @@ babel-eslint@10.1.0:
     eslint-visitor-keys "^1.0.0"
     resolve "^1.12.0"
 
+babel-jest@^28.1.3:
```
Comment (LiKang6688) [inline yarn.lock:2218]: Have you done 'yarn install' before commit?

Comment (Ashishyadav13) [reply yarn.lock:2218]: yes I did

**Pull request merged**